projects/storefrontlib/src/lib/product/components/product-list/product-view/product-view.component.scss

```diff
@@ -1,143 +1,97 @@
-        transform: rotate(90deg);
-        content: '';
-        position: absolute;
-        height: $cx-product-search-layout-grid-span-before-height;
-        width: $cx-product-search-layout-grid-span-before-width;
-        top: $cx-product-search-layout-grid-span-before-top;
-        left: $cx-product-search-layout-grid-span-before-left;
-        margin: $cx-product-search-layout-grid-span-before-trbl-margin;
-        @include var-color(
-          'background-color',
-          $cx-product-search-layout-grid-span-before-background-color
-        );
-      }
+    &:before {
+      -webkit-transform: rotate(90deg);
+      -moz-transform: rotate(90deg);
+      -ms-transform: rotate(90deg);
+      -o-transform: rotate(90deg);
+      transform: rotate(90deg);
+      content: '';
+      position: var(--cx-position, absolute);
+      width: var(--cx-width, 100%);
+      height: var(--cx-height, 2px);
+      top: var(--cx-top, 50%);
+      left: var(--cx-left, 0);
+      margin: var(--cx-margin, -1px 0 0 0);
+      background-color: var(--cx-g-color-inverse);
+    }
 
-      &:after {
-        -webkit-transform: rotate(0deg);
-        -moz-transform: rotate(0deg);
-        -ms-transform: rotate(0deg);
-        -o-transform: rotate(0deg);
-        transform: rotate(0deg);
-        content: '';
-        position: absolute;
-        height: $cx-product-search-layout-grid-span-before-height;
-        width: $cx-product-search-layout-grid-span-before-width;
-        top: $cx-product-search-layout-grid-span-before-top;
-        left: $cx-product-search-layout-grid-span-before-left;
-        margin: $cx-product-search-layout-grid-span-before-trbl-margin;
-        @include var-color(
-          'background-color',
-          $cx-product-search-layout-grid-span-before-background-color
-        );
-      }
+    &:after {
+      -webkit-transform: rotate(0deg);
+      -moz-transform: rotate(0deg);
+      -ms-transform: rotate(0deg);
+      -o-transform: rotate(0deg);
+      transform: rotate(0deg);
+      content: '';
+      position: var(--cx-position, absolute);
+
+      position: var(--cx-position, absolute);
+      width: var(--cx-width, 100%);
+      height: var(--cx-height, 2px);
+      top: var(--cx-top, 50%);
+      left: var(--cx-left, 0);
+      margin: var(--cx-margin, -1px 0 0 0);
+      background-color: var(--cx-g-color-inverse);
     }
   }
+}
 
-  &--list {
-    display: inline-block;
-    padding: $cx-product-search-layout-list-trbl-padding;
+.cx-product-list {
+  display: var(--cx-display, inline-block);
+  padding: var(--cx-padding, 13px 18px 13px 18px);
 
-    @include media-breakpoint-down(sm) {
-      padding: $cx-product-search-layout-list-mobile-trbl-padding;
-    }
+  @include media-breakpoint-down(sm) {
+    padding: var(--cx-padding, 13px 12px 13px 12px);
+  }
 
-    span {
-      @include var-color('color', $cx-product-search-layout-list-span-color);
+  span {
+    color: var(--cx-g-color-secondary);
```

Comment:
These two should be `color: var(--cx-color, --cx-g-color-secondary);` & `color: var(--cx-color, --cx-g-color-primary);`